### PR TITLE
Enable local development.

### PR DESCRIPTION
**Pull request by cmacfarl**
These changes allow one to run a flask server locally on a development machine.

Plus some git and gcloud ignore mods.

Longer term as we build out our cloud infrastructure we can look at serving static assets over a CDN, but it's not immediately clear what the benefit is of serving them out of buckets.  Although I'm open to be edumacated for that.

### CODE REVIEW

File: templates/labelVideo.html [hunk 1]
```diff
@@ -20,11 +20,8 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
-  <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
-  <script type="text/javascript"
-      src="https://storage.googleapis.com/{{project_id}}/compiled/js/fmltc.js?ignoreCache={{time_time}}">
-  </script>
+  <link rel="stylesheet" type="text/css" href="static/css/styles.css">
+  <script type="text/javascript" src="static/js/fmltc.js"> </script>
```
Comment (lizlooney) [inline templates/labelVideo.html:24]: static/js/fmltc.js doesn't exist, does it? Is there a change needed in scripts/deploy_js.sh?

File: templates/labelVideo.html [hunk 2]
```diff
@@ -20,11 +20,8 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
-  <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
-  <script type="text/javascript"
-      src="https://storage.googleapis.com/{{project_id}}/compiled/js/fmltc.js?ignoreCache={{time_time}}">
-  </script>
+  <link rel="stylesheet" type="text/css" href="static/css/styles.css">
```
Comment (lizlooney) [inline templates/labelVideo.html:23]: Does this work for quick changes to styles.css? I've had troubles with browsers caching the css and solved that by adding ?ignoreCache={{time_time}} to the url.

File: templates/login.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
   <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
   <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
+        href="/css/styles.css">
```
Comment (lizlooney) [inline templates/login.html:22]: This file has "/css/styles.css", but the other html files have "static/css/styles.css". Is that intentional?

File: templates/labelVideo.html (hunk 2 above)
Comment (cmacfarl) [reply templates/labelVideo.html:23]: I usually just do ctrl-f5 to flush the cache when I know I've changed css.  For ftc-scoring, when we push assets to the cloud we use a webpack cache busting technique to ensure that teams get the changes.  I'll look at that in more detail as we go.

File: templates/labelVideo.html (hunk 1 above)
Comment (cmacfarl) [reply templates/labelVideo.html:24]: To fully support this, there are changes needed to the build.   At present I'm just running the closure compile manually and copying the file over.  I'd work that out before removing the draft label on this pr.

**Comment by cmacfarl** [comment]
This work was subsumed into #78

**Pull request closed without merging**